pkgs/development/compilers/gmqcc/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, pkgs, ... }:
+pkgs.stdenv.mkDerivation rec {
+    pname = "gmqcc";
+    version = "297eab9";
+
+    src = pkgs.fetchgit {
+      url = "https://github.com/graphitemaster/gmqcc";
+      rev = "297eab9e5e2c9cc4f41201b68821593a5cf9a725";
+      sha256 = "1hl2qn7402ia03kjkblj4q4wfypxkil99sivsyk2vrnwwpdp4nzx";
+    };
+    #sourceRoot = "${pname}-${version}/Quake";
+
+    makefile = "Makefile";
+
+    installPhase = ''
+      runHook preInstall
```

Comment:
no need for this

this is sometimes required for some specific hooks

Reply:
I personally prefer to keep it.

Reply:
Since it doesn't install any other files and the hooks aren't used for anything, I've elected to remove those as well.

Reply:
Please keep the hooks. People sometimes needs to use `overrideAttrs` in packages, and ignoring the `preInstall`/`postInstall` hooks will cause the package to fail.

Also, if you eventually need to use some build hook, most of times they work by appending to the `pre`/`post` phases, and they will fail without error if you don't add it here.